src/components/Common/FieldsButton.js

```diff
@@ -95,15 +112,13 @@ export const FieldsButton = injector(
           overflow: 'auto',
         }}
       >
-        <Button
-          size={size}
-          icon={icon}
-          extra={trailingIcon}
-          style={style}
-          className={className}
-        >
-          {content.length ? content : null}
-        </Button>
+        {tooltip ? (
+          <Tooltip title={tooltip} theme={tooltipTheme}>
+            {renderButton()}
+          </Tooltip>
+        ) : (
+          renderButton()
+        )}
```

Comment:
I would be more reusable to use composition and not introduce this as a prop driven behaviour to this button. Is there any reason we couldn't just wrap the FieldsButton outside this definition?

Like this?

```tsx
<Tooltip title="Customize Columns" theme="dark">
  <Elem
    name="button-new"
    tag={FieldsButton}
    className={'newUi'}
    icon={<LsGearNewUI />}
    style={{ padding: 0 }}
    wrapper={FieldsButton.Checkbox}
  />
</Tooltip>
```

I always question introductions of components like this, it tends to not scale very well when we have to pass down props to orchestrate nested children that are not the main point to the initial component.